cmd/node/config/config.toml

```diff
@@ -36,6 +36,9 @@
    # MetaProtectionEnableEpoch represents the epoch when the transactions to the metachain are checked to have enough gas
    MetaProtectionEnableEpoch = 2
 
+   # AheadOfTimeGasUsageEnable represents the epoch when the cost of smart contract prepare changes from compiler per byte to ahead of time prepare per byte
+   AheadOfTimeGasUsageEnable = 2
```

Comment:
AheadOfTimeGasUsageEnableEpoch or AheadOfTimePrepareEnableEpoch or better shorter AheadOfTimeEnableEpoch here and in comment